### ISSUE TITLE
[Quest] Mark of the nexus king

### ISSUE THREAD

**Issue by RPAncap** [SunwellWoW/Sunwell-TBC-Bugtracker#4040]
Description: 
Hey I would like a confirmation that this Quest works. You are supposed to get an item that has 25% drop rate from an elite mob that spawns from prison in Bashir landing. They keys to open those prisons are pain to get. 

This quest is a part of a chain to summon YOR in heroic mana tombs.

I have already opened 16 of those prisons - the item didnt drop. Either this is bugged or I am extremely unlucky.

Name of the mobs : 

Trelopades<Legion Commander>
Solus the Eternal
Matron Li-sahar
King Dorfbruiser
Gorgolon the All-seeing

https://db.excalibur.ws/?item=31941#dropped-by:0-1-8

**Comment by Dootz** [comment]
Hi, currently they have various drop chance ( from 13 up to 24 ). With next server update it will be unified to 25%

**Comment by RPAncap** [comment]
"Riztazzdnes v 13:04
@here 
In 30 minutes from now on, i'll take nightbane offline for a quick update, realm will be down for around 5 minutes"

Does it mean resolved and live now ?

**Comment by Dootz** [comment]
Unfortunately fixes were not applied with today's restart. They should be with next update tomorrow.

**Comment by seaksd** [comment]
Hello, I have been on this quest for quite some time. In total 110x Ethereum prison keys turned into Ethereum Statis Chamber keys 22 times. I have opened and killed 1 of the 5 mobs inside 22 times now. Still have not seen a Mark of the Nexus-king. Including 8 of them today. It is possible I'm just extremely unlucky but statistically those are some really low odds. We also encountered 1 of the spawns Solus the Eternal on the south east Stasis chamber was invisible but we still killed and got the blue that dropped.
